.github/workflows/node.js.yml

```diff
@@ -44,11 +53,27 @@ jobs:
       - name: Install Prisma
         run: npx prisma generate
 
-      - run: npm run build --if-present
-      - run: npm test
-      - run: npm run integration-tests
+      - name: Build Backend
+        run: npm run build --if-present
+
+      - name: Verify Prisma Schema against Prisma Migrations
+        run: npm run db:verify-schema
```

Comment:
We now have a  CI step that validates the schema against the migration, so one cannot push invalid migrations anymore (or forget them)